release/models/system/openconfig-system-grpc.yang

```diff
@@ -101,9 +108,16 @@ module openconfig-system-grpc {
 
     leaf name {
       type string;
+      default "DEFAULT";
       description
         "The name of the gRPC server instance that is running on
-        the local system.";
+        the local system.
+
+        If the operator does not designate a name for the protocol
+        instance (e.g. config), the implementation should use the
+        name of 'DEFAULT' (e.g. state).  In addition, for
+        implementations that support a single gRPC server instance,
+        the default value is recommended for consistency.";
```

Comment:
Does this assume that there is a default gRPC server running on the system regardless of config at boot time?

Reply:
No - a `grpc-server` still needs to be instantiated by way of configuration.  A default value here does not imply the list member is always active with any downstream default/mandatory attributes

Default values on list keys are a bit unique because it must be defined to be instantiated so it does not solve that case

But rather it is similar to above in that:
* It is used as recommendation/guidance for clients to use to provision a single instance implementation (or the main list member) - driving consistency outside the model to *some* free-form string values where it may make sense
* It is used as a default name should a grpc-server endpoint be provisioned by way of non-OpenConfig (e.g. native) so that state context can be fulfilled to formulate a valid path/instance should a client still be interested in say subscribing to `/system/grpc-servers`

Reply:
SGTM